CHANGELOG.md

```diff
@@ -44,18 +47,16 @@ This release works with versions 0.7.\* of the PureScript compiler. It will not
 
 ## [v0.2.0-rc.1](https://github.com/purescript/purescript-semirings/releases/tag/v0.2.0-rc.1) - 2015-06-10
 
-Initial release candidate of the library intended for the 0.7 compiler.
+- Initial release candidate of the library intended for the 0.7 compiler.
 
 ## [v0.1.2](https://github.com/purescript/purescript-semirings/releases/tag/v0.1.2) - 2015-03-25
 
-More instances for `Free` (@cryogenian)
+- More instances for `Free` (@cryogenian)
 
 ## [v0.1.1](https://github.com/purescript/purescript-semirings/releases/tag/v0.1.1) - 2015-03-23
 
-Remove duplicated Bower file.
+- Remove duplicated Bower file.
 
 ## [v0.1.0](https://github.com/purescript/purescript-semirings/releases/tag/v0.1.0) - 2015-03-23
 
-Initial release:
-- Add free semiring
```

Comment:
This should probably be added again, but I'm not sure it makes much of a difference even if it is.